packages/visualizations/src/components/Map/MapRender.svelte

```diff
@@ -124,12 +148,15 @@ TODO:
     }
 
     $: updateStyle(style);
+
+    onDestroy(() => resizer?.disconnect());
+
 </script>
 
-<div id="map" bind:this={container} />
+<div id="map" bind:this={container} style={cssVarStyles} />
 
 <style>
     #map {
-        height: 400px;
+        aspect-ratio: var(--aspect-ratio);
```

Comment:
[According to caniuse](https://caniuse.com/?search=aspect-ratio), browser support for `aspect-ratio` is really fresh. We're not supposed to support older browsers, but considering how young it is, and that by default it will default to a height of 0 if I remember correctly, we could try to have something ugly but showing something at least.

Not knowing at first how to do that, I discovered https://www.stevefenton.co.uk/2021/04/check-css-support-in-css-using-supports-rules/. Ironically, it depends on something that isn't support by all browsers (`@supports`), but it's much much wider. We could fallback to a fixed height of `400px` for example if `aspect-ratio` isn't support maybe? It's still ugly though, but otherwise, apart from doing calculations in JS and losing the benefits of `aspect-ratio`...

Reply:
Indeed ! it's safer, I added an `@supports` fallback, the only case that won't be handled is if the browser doesn't support `aspect-ratio` and `@supports` in the same time...

I just wonder if something like this won't be even much safer :